### ISSUE TITLE
Need to run alternc.install twice

### ISSUE THREAD

**Issue by ulvida** [AlternC/AlternC#458]
When I install AlternC, I have to add Sid debian repository for phpmyadmin, and then solve dependencies of this pack. Il have to add previously (or manually go over proposed solutions bay aptitude) the following packages: 
- php-symfony-config
- php-symfony-dependency-injection
- php-symfony-expression-language
- php-symfony-service-contracts
- php-symfony-yaml

I don't know if there is a way to automate this. 
Moreover, after installing alternc and dependencies, when I run `alternc.insall` the first time, the installation is not completed. particualrly user admin/admon cannot login. I hve to run `alternc.insall` again, and then I can login.

**Comment by camlafit** [comment]
Hello

Alternc is not sid compliant, next 3.5 release could be supported buster no more.
And alternc don't use php-symfony-* packaging.

If i've misunderstood issue, you're welcome to reopened it :) Looks a duplicate to #417

**Comment by ulvida** [comment]
Hello Cam! Sorry, there were several things in my issue (I woul'd like to re-open it, but I can't): 

* Our [AlternC ansible role](https://github.com/UdelaRInterior/ansible-AlternC) is now buster compatible. When we felt on the absence of phpmyadmin in buster, I checked and there was no backport for phpmyadmin. So I installed phpmyadmin from Sid. the `php-synphony-*` modules (as well as [php-twig](https://github.com/UdelaRInterior/ansible-AlternC/issues/16)) are Sid dependencies of phpmyadmin. Thanks to your links, I saw now there is a backport, so [we will try it](https://github.com/UdelaRInterior/ansible-AlternC/issues/17). But this was secondary to my issue. The main is: 

* With the role, but also manually tested and re-tested, after installing alternc and dependencies, when I run `alternc.insall` the first time, the installation is not completed. particularly user admin/admon cannot login. I have to run `alternc.insall` again, and then I can login.
